llvm/docs/LangRef.rst

```diff
@@ -23624,9 +23624,9 @@ Examples:
       %active.lane.mask = call <4 x i1> @llvm.get.active.lane.mask.v4i1.i64(i64 %elem0, i64 429)
       %wide.masked.load = call <4 x i32> @llvm.masked.load.v4i32.p0v4i32(<4 x i32>* %3, i32 4, <4 x i1> %active.lane.mask, <4 x i32> poison)
 
-.. _int_experimental_get_alias_lane_mask:
+.. _int_experimental_get_nonalias_lane_mask:
 
-'``llvm.experimental.get.alias.lane.mask.*``' Intrinsics
+'``llvm.experimental.get.nonalias.lane.mask.*``' Intrinsics
```

Comment:
(Optional): What about `noalias` (currently there's 0 hits for "nonalias" in the langref, but "noalias" is a thing). 

```suggestion
'``llvm.experimental.get.noalias.lane.mask.*``' Intrinsics
```